test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2041,6 +2024,32 @@ class XdsEnd2endTest : public ::testing::TestWithParam<TestType> {
   grpc_channel_args xds_channel_args_;
 };
 
+AdsServiceImpl::AdsServiceImpl(bool enable_load_reporting)
```

Comment:
Please add a TODO assigned to me to move all of the code that deals with default resource contents out of `AdsServiceImpl` and into `XdsEnd2EndTest`.